docs/en/advanced_guides/datasets/kitti_det.md

```diff
@@ -87,48 +86,39 @@ kitti
 ```
 
```

Comment:
remove `Current tutorial is only for LiDAR-based and multi-modality 3D detection methods. Contents related to monocular methods will be supplemented afterwards.`